types.d.ts

```diff
@@ -1,23 +1,31 @@
 import { Boolean, Float, Integer, String, True } from "./alias";
 
 declare namespace Update {
+  /** Internal type holding properties that updates in channels share. */
   interface Channel {
+    chat: { type: Chat.ChannelChat["type"] };
     author_signature?: String;
-    chat: { type: "channel" };
-    from?: never;
+    from: never;
```

Comment:
It's impossible to create a value of type `never`, so this `interface` is impossible to instantiate!
```suggestion
    from?: never;
```